test/unit/facet_configuration_test.rb

```diff
@@ -7,15 +7,14 @@ module TestExtension
 end
 
 class FacetConfigurationTest < ActiveSupport::TestCase
-  class TestClass < HostFacets::Base
+  class TestFacet < HostFacets::Base
```

Comment:
There is easier way to solve this:
``` ruby
  class TestClass < HostFacets::Base
    self.table_name = 'test_facets'
  end
```

This sets proper table name for the model hence eliminating non-existing table error.

Reply:
> your solution is valid, but we could do it with less code lines.

Indeed, but I don't think it's the straight forward solution. The diff is larger, but we don't add more LOC with my proposal. And it's easier to read the code with my change imho.